src/app/util.cljs

```diff
@@ -0,0 +1,27 @@
+(ns app.util
+  (:require-macros [app.util])
```

Comment:
We need to add this, other wise our macro wont work.
https://code.thheller.com/blog/shadow-cljs/2019/10/12/clojurescript-macros.html